### DISCUSSION TITLE
editor: using arrow up / down to move between kiai or green lines

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#21376] [Ideas] [answered]
it just works so in stable so y not

upd: arrow up + arrow left -- move to next kiai / green / red line

**Comment by peppy** [comment, marked as answer]
I have no idea what this means.

**Reply by Dopaminos** [reply]
when you are at the beginning of the map timeline, in stable editor, you can press the up arrow + left arrow to quickly move to next kiai or green/red line

lazer doesn't have this so... yea.

**Reply by peppy** [reply]
Thanks, I understand now.

**Reply by peppy** [reply]
#21387
